ironfish-cli/src/commands/wallet/send.ts

```diff
@@ -255,6 +254,8 @@ export class Send extends IronfishCommand {
       raw = RawTransactionSerde.deserialize(bytes)
     }
 
+    displayTransactionSummary(raw, assetData, amount, from, to, memo)
+
```

Comment:
Makes sens to show the summary before exiting for the rawTransaction/ unsignedTransaction flags

Reply:
Agreed - big improvement. Small nit though, can you add a newline to the end of the transaction summary output to improve the readability between the summary and the raw/unsigned tx output?